lib/launcher.js

```diff
@@ -217,12 +217,13 @@ class Launcher {
             this.logBuffer.add({ level: 'system', msg: 'Updating project dependencies' })
             pkg.dependencies = wantedDependencies
             fs.writeFileSync(pkgFilePath, JSON.stringify(pkg, null, 2))
+            const npmEnv = this.settings.env
```

Comment:
Ben, `settings.env` normally only contains things like FF_INSTANCE_ID etc

Should we not clone `process.env` and add *PROXY* vars from `settings`? 

See previous comment.